solutions/0300-0399/0312-burst-balloons-hard.md

```diff
@@ -53,6 +53,8 @@ We can see the transformation equation is very similar to the one for matrix mul
 
 `dp[i][j] = max(dp[i][j], nums[i] * nums[k] * nums[j] + dp[i][k] + dp[k][j]) # i < k < j`
 
+`dp[i][j]` is the coins obtained from bursting all balloons between left and right (not including left or right)
```

Comment:
```
Make sure you've applied LaTex for variables, formulas and time / space complexity instead of using backticks
```

You keep using backticks while i'm asking for LaTex.

This is backtick.

````
`dp[i][j]`
````

which will be rendered as `dp[i][j]`.

This is LaTex

````
$dp[i][j]$
````

which will be rendered as $dp[i][j]$.

See the difference now?